.tekton/retag-collector-full.yaml

```diff
@@ -13,10 +13,7 @@ metadata:
       (event == "push" && target_branch.matches("^(master|release-.*)$")) ||
       (event == "pull_request" && (source_branch.matches("(konflux|renovate|appstudio|rhtap)") || body.pull_request.labels.exists(l, l.name == "konflux-build")))
   labels:
```

Comment:
I assume leaving this empty `labels:` line in was an intentional choice, which is okay with me, but wondering if you think it is nicer to remove it.

Reply:
Do not feel strongly about it and did not actually put much thought. Left it to highlight that unlike other `Pipeline`-s in the repo, these aren't component pipelines (because component relationship is set in labels).